assets/apps/dashboard/src/Components/Content/Start.js

```diff
@@ -132,6 +133,40 @@ const Start = (props) => {
 					</Card>
 				</>
 			)}
+			{!pro && (
+				<>
+					<Card
+						classNames="woo-card"
+						icon="cart"
+						dashicon={true}
+						lockIcon={true}
+						title={__('WooCommerce Booster', 'neve')}
+						description={__(
+							'Empower your online store with awesome new features, specially designed for smooth WooCommerce integration.',
```

Comment:
This string is slightly different from the one we already use in the free vs pro tab. 🙈  Let's change it to 

```suggestion
							'Empower your online store with awesome new features, specially designed for a smooth WooCommerce integration.',
```